cli_tools/common/utils/logging/service/literal_loggable.go

```diff
@@ -17,18 +17,23 @@ package service
 type literalLoggable struct {
 	strings   map[string]string
 	int64s    map[string][]int64
+	bools     map[string]bool
 	traceLogs []string
 }
 
 func (w literalLoggable) GetValue(key string) string { return w.strings[key] }
 
+func (w literalLoggable) GetValueAsBool(key string) bool { return w.bools[key] }
+
 func (w literalLoggable) GetValueAsInt64Slice(key string) []int64 { return w.int64s[key] }
 
 func (w literalLoggable) ReadSerialPortLogs() []string { return w.traceLogs }
 
 // SingleImageImportLoggable returns a Loggable that is pre-initialized with the metadata
 // fields that are relevant when importing a single image file.
-func SingleImageImportLoggable(fileFormat string, sourceSize, resultSize int64, matchResult string, inflationTypeStr string, inflationTimeInt64 int64, shadowInflationTimeInt64 int64, traceLogs []string) Loggable {
+func SingleImageImportLoggable(fileFormat string, sourceSize, resultSize int64, matchResult string,
+	inflationTypeStr string, inflationTimeInt64 int64, shadowInflationTimeInt64 int64,
+	isUEFICompatibleImageBool bool, isUEFIDetectedBool bool, traceLogs []string) Loggable {
```

Comment:
Consider refactoring to a builder as 10+ args is quite a lot for a func. Easy to make a mistake as well.